cmd/Makefile.am

```diff
@@ -128,14 +132,23 @@ noinst_PROGRAMS += decode-mount-opts/decode-mount-opts
 decode_mount_opts_decode_mount_opts_SOURCES = \
 	decode-mount-opts/decode-mount-opts.c
 decode_mount_opts_decode_mount_opts_LDADD = libsnap-confine-private.a
+decode_mount_opts_decode_mount_opts_STATIC =
+
+if STATIC_LIBCAP
+decode_mount_opts_decode_mount_opts_STATIC += -lcap
+else
+decode_mount_opts_decode_mount_opts_LDADD += -lcap
```

Comment:
Sorry for not suggesting this earlier, but wouldn't it be simpler to do this:
```
if STATIC_LIBPCAP
decode_mount_opts_decode_mount_opts_LDADD += libcap.a
else
decode_mount_opts_decode_mount_opts_LDADD += libcap.a
endif
```

Wouldn't that avoid the further fiddling with "hacked rules" described below?